src/PrestaShopBundle/Controller/Admin/ProductController.php

```diff
@@ -674,6 +674,45 @@ public function unitAction(Request $request, $action, $id)
         }
     }
 
+    public function exportAction($_format)
+    {
+        // init vars
+        $productProvider = $this->container->get('prestashop.core.admin.data_provider.product_interface');
+        /* @var $productProvider ProductInterfaceProvider */
```

Comment:
Can you reverse the PHPDoc annotation and the variable declaration ?

```php
/* @var $productProvider ProductInterfaceProvider */
$productProvider = $this->container->get('prestashop.core.admin.data_provider.product_interface');
```

I'm not sure the parsed information is valid this way...